qiskit/ignis/experiments/calibration/methods/basic_1q.py

```diff
@@ -54,29 +50,23 @@ def __init__(self,
             pulse_name: Pulse name in the database entry to provide parameter set to
                 construct pulse schedule to calibrate. By default pi pulse parameter is used.
         """
-        param_dict = table.get_instruction_kwargs(
-            qubits=qubit,
-            channel='d*',
-            inst_name=pulse_name
-        )
 
         # todo get qubit property from other database.
         # channel ref frequency is different from pulse sideband and thus
         # this value is stored in another relational database.
         # something like qubit property table where f01, anharmonicity, T1, T2, etc... exist.
-        freq01 = circuit.Parameter('q{ind}.d{ind}.f01'.format(ind=qubit))
+        freq01 = 0.0
```

Comment:
I remember we also need to address pulse phase/frequency issue (currently we don't have framework to manage them). note for reminder.

Reply:
You are correct. I would see this a subsequent PR.